rest_framework/filters.py

```diff
@@ -100,13 +101,17 @@ def filter_queryset(self, request, queryset, view):
 
         orm_lookups = [self.construct_search(six.text_type(search_field))
                        for search_field in search_fields]
-
+        and_queries = []
```

Comment:
I don't sure, if calling `queryset.filter(pk_in=set(pk_list))` will make duplications go away, as they `pk` still will be in `pk_list`. I want to filter original `queryset` by `pk_list`.
When we call `filter` with multiple kwargs or several times on one queryset it is equal to logical `and`.